test/luatest_test.lua

```diff
@@ -79,13 +79,31 @@ g.test_skip_if_tnt_specific = function()
     t.assert_equals(helper.assert_failure(t.skip_if, {}, 'expected').status, 'skip')
 end
 
+g.test_skip_no_reason_specified = function()
+    local result = helper.assert_failure(t.skip)
+    t.assert_equals(result.status, 'skip')
+    t.assert_equals(result.message, 'No reason')
+end
+
+g.test_skip_if_no_reason_specified = function()
+    local result = helper.assert_failure(t.skip_if, true)
+    t.assert_equals(result.status, 'skip')
+    t.assert_equals(result.message, 'No reason')
+end
+
+g.test_run_only_if_no_reason_specified = function()
+    local result = helper.assert_failure(t.run_only_if, false)
+    t.assert_equals(result.status, 'skip')
+    t.assert_equals(result.message, 'No reason')
+end
+
 g.test_success_if_tnt_specific = function()
     assert_any_error(t.success_if, box.NULL)
     t.assert_equals(helper.assert_failure(t.success_if, true).status, 'success')
     t.assert_equals(helper.assert_failure(t.success_if, {}).status, 'success')
 end
 
-g.test_assert_aliases = function ()
+g.test_assert_aliases = function()
```

Comment:
It seems the formatting fix is not related to the task, I would suggest discarding this change to make the change history clean.

Reply:
Ok, fixed.